builtin/commands/dposV2_commands.go

```diff
@@ -98,14 +98,18 @@ func CheckDelegationCmdV2() *cobra.Command {
 	return &cobra.Command{
 		Use:   "check_delegationV2 [validator address]",
 		Short: "check delegation to a particular validator",
-		Args:  cobra.MinimumNArgs(1),
+		Args:  cobra.MinimumNArgs(2),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			var resp dposv2.CheckDelegationResponseV2
-			addr, err := cli.ResolveAddress(args[0])
+			validatorAddress, err := cli.ResolveAddress(args[0])
```

Comment:
Should use util.ParseAddress here, ResolveAddress only makes sense for contract names/addresses.

Reply:
addressed in 22b1267